tests/mail/tests.py

```diff
@@ -894,6 +894,21 @@ def test_dont_mangle_from_in_body(self):
         )
         self.assertNotIn(b">From the future", email.message().as_bytes())
 
+    def test_long_address_folding(self):
+        # Ticket #35378
+        # should encode long addresses with special characters using
+        # 7bit Content-Transfer-Encoding
```

Comment:
We no longer add comments in the tests until its really important. So maybe these can be removed.